### DISCUSSION TITLE
Multi auth feature for filament

### DISCUSSION THREAD

**Discussion by mnafullstack** [filamentphp/filament#3196] [Ideas]
There should be a multi-auth feature for filament users because it is very important for developers. I have searched a lot but could not find a way to use multi-auth with filament. I have also tried to use roles and permissions with the filament shield but the filament shield is not getting installed with the filament user management package because of a version conflict issue.

If I could use the filament shield with the filament user manager package, this would be a great help for me to create a normal customer user and an admin by assigning permissions for pages to both of them. Is there still any kind of thing from which I can achieve the things I want?

**Comment by vv-ac** [comment]
Hi @drifteaur, do you mean MFA?
If so, here you go: https://filamentphp.com/plugins/breezy

**Comment by danharrin** [comment]
https://github.com/artificertech/filament-multi-context
